pages/FormPage/index.js

```diff
@@ -0,0 +1,5 @@
+import Form from "@/components/Form/index.js";
+
+export default function AddPlacesPage() {
+    return <Form></Form>
```

Comment:
```suggestion
    return <Form />
```
If you don't intend on passing children, you may self-close the component.

Reply:
Thanks! I changed that.